__tests__/index.test.jsx

```diff
@@ -6,7 +6,7 @@
 
 import React from "react";
 import { render, screen } from "@testing-library/react";
-import Home from "../pages/index";
+import Home from "../pages/Index";
```

Comment:
```suggestion
import Home from "../pages/index";
```
was correct before ;)